packages/esm-patient-label-printing-app/src/config-schema.ts

```diff
@@ -0,0 +1,13 @@
+import { Type } from '@openmrs/esm-framework';
+
+export const configSchema = {
+  showPrintIdentifierStickerButton: {
+    _type: Type.Boolean,
+    _description: "Whether to display the 'Print identifier sticker' button in the patient banner",
+    _default: true,
```

Comment:
```suggestion
    _default: false,
```

Maybe this should default to `false` so implementations can choose to opt-in via config.